app/views/DisplayPrefsView.tsx

```diff
@@ -16,12 +16,29 @@ import SafeAreaView from '../containers/SafeAreaView';
 import { ICON_SIZE } from '../containers/List/constants';
 import log, { events, logEvent } from '../utils/log';
 import { DISPLAY_MODE_CONDENSED, DISPLAY_MODE_EXPANDED } from '../constants/constantDisplayMode';
-
-const DisplayPrefsView = props => {
+import { SettingsStackParamList } from '../stacks/types';
+
+interface IParam {
+	sortBy?: 'alphabetical' | 'activity';
```

Comment:
What do you think if we create types for sortBy and displayMode, for example: 

`type TSortBy = 'alphabetical' | 'activity';` 
`type TdisplayMode = typeof DISPLAY_MODE_EXPANDED | typeof DISPLAY_MODE_CONDENSED;`